From d7dd25a26d67bdae8d1e43240151fdce4395a427 Mon Sep 17 00:00:00 2001
From: Chris Park <Chris.Park@amd.com>
Date: Mon, 4 Jan 2016 16:42:22 -0500
Subject: [PATCH 0656/1110] drm/amd/dal: Fix bug when cleaning up dc->ctx

Since free dc clears the memory block for dc including dc->ctx, no need to clear dc->ctx again.

Signed-off-by: Chris Park <Chris.Park@amd.com>
Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/core/dc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/core/dc.c b/drivers/gpu/drm/amd/dal/dc/core/dc.c
index df243c2..ea31bc6 100644
--- a/drivers/gpu/drm/amd/dal/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/dal/dc/core/dc.c
@@ -402,7 +402,6 @@ void dc_destroy(struct dc **dc)
 {
 	destruct(*dc);
 	dc_service_free((*dc)->ctx, *dc);
-	dc_service_free((*dc)->ctx, (*dc)->ctx);
 	*dc = NULL;
 }
 
-- 
2.7.4

